sys/aarch64/pmap.c

```diff
@@ -77,7 +77,7 @@ pte_t pte_make(paddr_t pa, vm_prot_t prot, unsigned flags) {
 
 pte_t pte_protect(pte_t pte, vm_prot_t prot) {
   return vm_prot_map[prot] |
-         (pte & (~ATTR_AP_MASK & ~ATTR_XN & ~ATTR_SW_FLAGS)) |
+         (pte & (~ATTR_AP_MASK & ~ATTR_XN & ~ATTR_SW_FLAGS & ~ATTR_AF)) |
```

Comment:
`pte & ~(ATTR_AP_MASK | ATTR_XN | ATTR_SW_FLAGS | ATTR_AF)` is more readable

Reply:
done.